### PR TITLE
fix(RHTAPBUGS-933): create-pyxis-image tests

### PR DESCRIPTION
**Pull request by mmalina**
Some of the tests were creating wrong sample data. Also, I added a few checks.

### CODE REVIEW

File: tasks/create-pyxis-image/tests/test-create-pyxis-image-multiple-containerimages.yaml [hunk 1]
```diff
@@ -82,5 +85,32 @@ spec:
                 cat $(workspaces.data.path)/mock_create_container_image.txt
                 exit 1
               fi
+
+              if [ $(cat $(workspaces.data.path)/mock_skopeo.txt | wc -l) != 6 ]; then
```
Comment (johnbieren) [inline tasks/create-pyxis-image/tests/test-create-pyxis-image-multiple-containerimages.yaml:89]: Isn't this already checked (but more thoroughly) by the code below?

Comment (mmalina) [reply tasks/create-pyxis-image/tests/test-create-pyxis-image-multiple-containerimages.yaml]: yeah, ok, removed

**Review by johnbieren — APPROVED** [review]

**Review by johnbieren — APPROVED** [review]
I wonder if the `test` prefix was more appropriate than the `fix` prefix here. But if it is resolving a bug, maybe `fix` was the way to go. Not asking for a change, just thinking out loud

**Comment by mmalina** [comment]
> I wonder if the `test` prefix was more appropriate than the `fix` prefix here. But if it is resolving a bug, maybe `fix` was the way to go. Not asking for a change, just thinking out loud

Good question. I wasn't really aware of `test`, so good to know. But yeah, I'd say the tests were a bit wrong, so I'd still consider this to be a fix.

**Pull request merged**